pykickstart/sections.py

```diff
@@ -126,6 +126,8 @@ def __init__(self, *args, **kwargs):
 class ScriptSection(Section):
     allLines = True
     description = ""
+    epilog = ""
+    title = ""
```

Comment:
These three (description, epilog, and title) should start with an underscore if it's expected they will not be part of the public API that users of pykickstart can do something with.  To me, they very much feel like internal-only things.